### ISSUE TITLE
Improving dependencies

### ISSUE THREAD

**Issue by HugoGiraudel** [SaraVieira/react-social-sharing#16]
Hey Sara! 👋 

1. I see this package has `jest` as a (non-dev) dependency. Is this expected?
2. Could we add `react` as a peer dependency?

**Comment by SaraVieira** [comment]
Two very good points! I'll set up my local enviroment today since I just changed computers and have this done in like an hour! Thank you :)

**Comment by HugoGiraudel** [comment]
Thaaaaanks! ✨ 

I think I might create a pull request to make it possible to localise the `aria-label` at some point. :)

**Comment by HugoGiraudel** [comment]
Thanks for the quick fix Sara. :)

For the translation, we could have a function prop:

```jsx
<Twitter
  link="https://github.com"
  label={serviceName => 'Share on ' + serviceName}
/>
```

**Comment by SaraVieira** [comment]
But that would mean that the "Share on" part would not be localized :(

**Comment by HugoGiraudel** [comment]
No, that’s the entire point, it would. Let me clarify my example. This is your current implementation:

https://github.com/SaraVieira/react-social-sharing/blob/be8cb254ba3ee4b151c64c8465e91e3aef7f0b89/src/buttons/factory.js#L23-L31

On line 29 and 30, instead of doing `Share on ${ariaName || name}`, you would do something like:

```jsx
typeof label === 'function' // Or the name you want for the prop
  ? label(ariaName || name)
  : 'Share on ${ariaName || name}
```

Which would allow a user to have something like:

```jsx
<Twitter
  link="https://github.com"
  label={serviceName => localise('share_label', { serviceName })}
/>
```

With the English version of `share_label` being something like “Share that link on {{ serviceName }}” or whatever.

Basically, if a function is being given to you, you call it with the service name, otherwise you default to English. Does it make sense?

**Comment by SaraVieira** [comment]
I am so sorry this completly slipped my notifications :(

It has been added in https://github.com/SaraVieira/react-social-sharing/releases/tag/v3.0.0
